README.md

```diff
@@ -242,6 +242,10 @@ Options:
                                   for multi-granule files). Beware that multi-
                                   granule datasets without a granule id in the
                                   filename will overwrite each-other
+  --throughly-check-existing / --cheaply-check-existing
+                                  Should we open every dataset to check if
+                                  *all* inner granules have been produced?
+                                  Default: false.
```

Comment:
*all* inner granules have been produced?
Needed for pre-2010(?) multi-granule data.
                                  Default: false.

Reply:
It's not _strictly_ needed for pre-2016 data, because it will still run through all of the inner granules when processing that data.

This only affects the skipping of datasets when scanning for things to package. It wont check whether _all_ of the granules individually are skippable. If one is visible, it assumes they're all completed for that zip file.